e2e/src/committer.rs

```diff
@@ -132,6 +131,9 @@ impl Committer {
                 "COMMITTER__APP__FEE_ALGO__ALWAYS_ACCEPTABLE_FEE",
                 u64::MAX.to_string(),
             )
+            .env("COMMITTER__APP__TX_FEES__MAX", "4000000000000000")
+            .env("COMMITTER__APP__TX_FEES__MIN_REWARD_PERC", "20")
+            .env("COMMITTER__APP__TX_FEES__MAX_REWARD_PERC", "30")
```

Comment:
We should mention the new params to the docs

Reply:
[4214585](https://github.com/FuelLabs/fuel-block-committer/commit/4214585f123beea0b9852d1eeb8e1e10e04fc208)